src/code/index.tsx

```diff
@@ -0,0 +1,142 @@
+import { showOpenOrCreateDialog } from "./components/open-or-create";
+import { showAbout } from "./components/about";
+import { codapIframeSrc } from "./codap-iframe-src";
+// CFM is added using script tag.
+const CloudFileManager = (window as any).CloudFileManager;
+
+import "../styles/index.sass";
+
+const getQueryVariable = name => {
```

Comment:
This can be replaced with the `query-string` package already used elsewhere.

Reply:
Right, that's what I meant by:

> I'll be able to reuse query-parameter helpers

I can do it now too.